web/src/components/SearchBox.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
@dchiquit I put this here in case you find it helpful, but feel free to delete as part of #182